cnv/cnv_users_guide/cnv-importing-virtual-machine-images-datavolumes.adoc

```diff
@@ -0,0 +1,28 @@
+[id="cnv-importing-virtual-machine-images-datavolumes"]
+= Importing virtual machine images with DataVolumes
+include::modules/cnv-document-attributes.adoc[]
+:context: cnv-importing-virtual-machine-images-datavolumes
+toc::[]
+
+You can import an existing virtual machine image into your {product-title}
+cluster. {ProductName} uses DataVolumes to automate the import of data and the
+creation of an underlying PersistentVolumeClaim.
```

Comment:
Just a suggestion, feel free to ignore.

You could add `(PVC)` after the first declaration of PersistentVolumeclaim here, and then use the abbreviation in the caution block.